data/core/command.lua

```diff
@@ -98,12 +141,25 @@ local function perform(name, ...)
 end
 
 
+---Performs a command.
+---
+---The arguments passed into this function is forwarded to the predicate function. </br>
+---If the predicate function returns more than 1 value, the second to last values
+---are passed to the function associated to the command.
```

Comment:
Mmmh, this is a bit convoluted. What about:
```
If the predicate returns more than 1 value, the other values are passed to the command.
```

Reply:
LGTM.